FrameworkSystem/private/standardLogging/Logging.py

```diff
@@ -0,0 +1,388 @@
+
+  Logging could delegate the initialization and the configuration to a factory of the root logger be it can not
+  because it has to wrap the old gLogger.  
+
+  Logging should not be instancied directly. It is LoggingRoot which is instancied and which instantiates Logging
+  objects.
+  """
+
+  # componentName is a class variable because the component name is the same for every Logging objects
+  # its default value is "Framework" but it can be configured in initialize() in LoggingRoot
+  # it can be composed by the system name and the component name. For instance: "Monitoring/Atom"
+  _componentName = "Framework"
+
+  # all the different backends
+  _BACKENDSDICT = {'stdout': StdoutBackend,
+                   'stderr': StderrBackend,
+                   'file': FileBackend,
+                   'server': RemoteBackend}
+
+  def __init__(self, father=None, fatherName='', name=''):
+    """
+    Initialization of the Logging object.
+    :params father: Logging, father of this new Logging.
+    :params fatherName: string representing the name of the father logger in the chain.
+    :params name: string representing the name of the logger in the chain. 
+    By default, 'fatherName' and 'name' are empty, because getChild accepts only string and the first empty
+    string corresponds to the root logger. 
+
+    Example: 
+    logging.getLogger('') == logging.getLogger('root') == root logger
+    logging.getLogger('root').getChild('log') == root.log == log child of root
+    """
+
+    # Logging chain
+    self._children = {}
+    self._parent = father
+
+    # initialize display options and level with the ones of the Logging parent
+    if self._parent is not None:
+      self._options = self._parent.getDisplayOptions()
+      self._level = LogLevels.getLevelValue(father.getLevel())
+    else:
+      self._options = {'showHeaders': True, 'showThreads': False, 'Color': False, 'Path': False}
+      # the native level is not used because it has to be to debug to send all messages to the log central
+      self._level = None
+
+    # dictionary of the option state, modified by the user or not
+    # this is to give to the options the same behaviour that the "logging" level:
+    # - propagation from the parent to the children when their levels are not set by the developer himself
+    # - stop the propagation when a developer set a level to a child
+    self._optionsModified = {'showHeaders': False, 'showThreads': False}
+    self._levelModified = False
+
+    self._backendsList = []
+
+    self._logger = logging.getLogger(fatherName).getChild(name)
+
+    # name of the Logging
+    self.name = name
+
+    # entire name of the Logging based on the logger name of 'logging': this name is the entire name with parent names:
+    # 'root' representing the root logger name is removed
+    # all "." separator are replaced by "/"
+    self.entireName = self._logger.name.replace("root", "")
+    self.entireName = self.entireName.replace(".", "/")
```

Comment:
Okay great. I just went through the code of logging, and there is no way to change the dot indeed. 
However, I recommend that you send an email to the mailing list, asking what they think about having a dot instead of a slash. Maybe it simplifies our life.
Btw, note that with this technique, you have a problem if the logger is called "a.b.c" no ?